programs/ssl/ssl_client2.c

```diff
@@ -2597,7 +2750,74 @@ int main( int argc, char *argv[] )
         }
         mbedtls_printf("\n");
     }
-#endif
+
+#if defined( MBEDTLS_SSL_DTLS_SRTP )
+    else if( opt.use_srtp != 0  )
+    {
+        size_t j = 0;
+        mbedtls_dtls_srtp_info dtls_srtp_negotiation_result;
+        mbedtls_ssl_get_dtls_srtp_negotiation_result( &ssl, &dtls_srtp_negotiation_result );
+
+        if( ( dtls_srtp_negotiation_result.chosen_dtls_srtp_profile
+                                == MBEDTLS_TLS_SRTP_UNSET ) )
```

Comment:
The CI complains about the extra parentheses here.

Reply:
thanks, was not able to access the CI logs to see what it was complaining about. I just pushed the fix,

Reply:
Yeah, we've been wanting to make the CI logs public for a long time, but it looks like that we still won't be able to do it in the near future :( 

Thank you for the fix!